src/backend/functions/projects.ts

```diff
@@ -127,8 +128,7 @@ const projectTransformer = (
     goals: project.goals.map(descBulletConverter),
     duration: project.workPackages.reduce((prev, curr) => prev + curr.duration, 0),
     workPackages: project.workPackages.map((workPackage) => {
-      const endDate = new Date(workPackage.startDate);
-      endDate.setDate(workPackage.duration * 7);
+      const endDate = calculateEndDate(workPackage.startDate, workPackage.duration);
```

Comment:
Since this variable is just declared and used once as part of the returned object, I think it can go directly in the return. This also applies to the other backend function for projects